CMakeLists.txt

```diff
@@ -68,3 +75,6 @@ endif ()
 #    DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/bin/hello_2
 #    WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
 #)
+
+
+add_subdirectory(test)
```

Comment:
This line should be inside the if (BUILD_TESTS) block

Reply:
I made the new changes in the 'dev' branch, I think you don't have to pull request for this file.